src/amo/components/AddAddonToCollection/index.js

```diff
@@ -290,10 +290,7 @@ export class AddAddonToCollectionBase extends React.Component<InternalProps> {
   }
 }
 
-export const mapStateToProps = (
-  state: AppState,
-  ownProps: Props,
-): PropsFromState => {
+const mapStateToProps = (state: AppState, ownProps: Props): PropsFromState => {
```

Comment:
We no longer need to export `mapStateToProps` because it isn't being used in a test.